src/index.js

```diff
@@ -29,6 +29,15 @@ const client = new Discord.Client({ partials: ["MESSAGE", "REACTION"] });
 
 const users = {}; // Message counter object for users
 const chnlMsgs = {}; // Separate message counter object unrelated to users
+const isMessageIgnored = (message) => {
```

Comment:
If it isn't too much trouble could this be changed to 
```suggestion
function isMessageIgnored(message) {
```
to match the way functions have been defined elsewhere in the repository